pkg/storaged/crypto-tab.jsx

```diff
@@ -57,6 +56,7 @@ var CryptoTab = React.createClass({
                 function (items) {
                     old_config = utils.array_find(items, function (c) { return c[0] == "crypttab"; });
                     new_config = [ "crypttab", old_config ? $.extend({ }, old_config[1]) : { } ];
+                    new_config = [ "crypttab", old_config ? Object.assign({ }, old_config[1]) : { } ];
```

Comment:
I think you meant to replace the previous line, not just add this new one.

Reply:
Ah, already fixed in the last commit, so fine. Therefore, just the question left.

Reply:
Indeed!  The change is wrongly split across two commits.